client/src/legacy/createEvent.js

```diff
@@ -0,0 +1,20 @@
+/* global window */
+// Polyfill IE11
```

Comment:
this doesn't seem like it's a polyfill, might be better to remove this comment and put a comment in the `new Event(type)` block that this is for IE11

Reply:
Changed comment to state that it is a helper function instead.